src/utils/format.ts

```diff
@@ -59,3 +59,34 @@ export const toUpperCase = (str: string | undefined): string => {
 export const equalsIgnoreCase = (compareTo: string, compare: string): boolean => {
   return compare.localeCompare(compareTo, LOCALE_EN, { sensitivity: 'accent' }) === 0;
 };
+
+export const retrieveDirectorNameFromAppointment = (appointment: CompanyAppointment ): string => {
+  if (appointment.forename && appointment.otherForenames && appointment.surname) {
```

Comment:
Do we want to return white spaced strings if any? Else we may have to handle those as well.

Reply:
Hello, not sure what you are referring to. Do you mean a string that could be a white space, or whether the fields need to be trimmed incase they contain white space?

Reply:
yes trimming the string containing white spaces would work. if condition on an empty string "" will not run the code but  will run if it is made up of at-least a white space. Hence, forename, other-name or surname could display nothing. If this is handled by the form validation then that is fine. Just mentioned it in passing otherwise everything looks good.